### DISCUSSION TITLE
Websites where we can list our PyCon India 2024 event

### DISCUSSION THREAD

**Discussion by tankala** [pythonindia/inpycon2024-tasks#28] [General]
Please share the Website or Calendar where we can list our PyCon India 2024 event.

If you know the procedure or person whom we need to contact for listing please mention those details too.

This will help our PyCon India 2024 event reach more people

**Comment by anistark** [comment]
Already listed on [PyCon APAC](https://pycon.asia/#events). Thanks to @vellankikoti

**Comment by anistark** [comment]
- [x] [Python Events Calendar](https://www.python.org/events/).
- [x] [Dev Events](https://dev.events/python)
- [x] [confs.tech](https://confs.tech/python)
- [ ] [eventy.co](https://www.eventyco.com/events/conferences/tech~python)
- [x] [python-organizers](https://github.com/python-organizers/conferences/blob/main/2024.csv)

Please also research more places to share.

**Reply by tankala** [reply]
Done at Dev Events
Done at confs.tech
Done at python-organizers
Done at eventy.co
Added to https://pycon.org/#calendar
Need to be added at https://www.python.org/events/. Mailed them again

**Comment by tankala** [comment]
we need to create an event on LinkedIn, Facebook, and Meetup
- [ ] LinkedIn
- [ ] Facebook
- [ ] Meetup
- [x] https://pythondeadlin.es/
- [x] https://github.com/scraly/developers-conferences-agenda - developers.events

**Reply by anistark** [reply]
Need to get banners and content for this.

**Reply by tankala** [reply]
I will check with Akashnil & Content team regarding this. We are planning to do it once tickets are available so that it can jell together.

**Comment by tankala** [comment]
- [ ] Approach popular cities communities to create an event in their meetup group, social media & website for PyCon India 2024 Conference
- [ ] Reaching out to institutions/companies for promotion

**Reply by anistark** [reply]
Personal opinion: We shouldn't do institutions/companies promotions. It'll start to seem like they've sponsored us in-kind.

**Reply by tankala** [reply]
Got it. Make sense. What about colleges and universities?

**Reply by anistark** [reply]
We shouldn't ask them to do direct promotions. 
We had Python Express. A program where we go to colleges/universities to organise workshops, which we can promote and in turn can reflect upon PyCon India.
